pages/windows/taskkill.md

```diff
@@ -0,0 +1,27 @@
+# taskkill
+
+> Terminate a process by its process id or name.
+
+- Display information about the usage of the command:
+
+`taskkill /?`
+
+- Terminate a process by its id:
+
+`taskkill /pid {{process_id}}`
+
+- Terminate a process by its name:
+
+`taskkill /im {{process_name}}`
+
+- Forcefully terminate a specified process:
+
+`taskkill /f`
+
+- Terminate a process and its child processes:
+
+`taskkill /t`
```

Comment:
Perhaps we could add `/pid {{process_id}}` to this or something for the same reason?